printPrimeNumbers.js

```diff
@@ -0,0 +1,18 @@
+let isPrime = true;
+ 
+let i =2;
+while(i <= 100){
+  let j=2;
+  while(j < i - 1){
+    if(i % j == 0){
+      isPrime = false;
+      break;
+    }
+    j++;
+  }
+  if(isPrime){
+    console.log(i);
+  }
+  isPrime = true;
+  i++;
+}
```

Comment:
Too many `whiles`, this is not optimal.

- Use `Array.from` to create an array of numbers from 1 to 100
- Use `Array.filter` and the `isPrime` function you created earlier to remove everyone that is not prime.

Reply:
fixed on 373c405